sources/T-Editing-Guidelines-201602.adoc

```diff
@@ -0,0 +1,843 @@
+= Author's guide for drafting ITU-T Recommendations
+ITU
```

Comment:
@anermina why is there an "ITU" line here? Let's remove it.

Reply:
I'm not sure why this was written in G.650 and Y.3507 documents, which I used as a template. Will remove this line.

Reply:
Thanks, we should also remove it from those two files.

Reply:
_Do not. Do not remove it. Do not._

Random things break in the processing of Asciidoctor document attributes if you leave out the author name. I do not remember what they are, but there is no reason to risk them breaking just because you don't think it looks neat. 

@manuel489: this also applies to the templates you are editing. 

I'm editing https://www.metanorma.com/author/topics/document-format/meta-attributes/ to reflect this.

Reply:
@opoudjis Unfortunately you're wrong here. The author name is only optional in AsciiDoc. There is no issue with leaving out the author name. And in this case, the name is wrong anyway. If you have been keeping up with the sample documents and templates -- ALL mn-samples-* documents DO NOT HAVE THIS FIELD. Gosh. Remove this already.

Reply:
@anermina @manuel489 please remove these lines because they are wrong and misleading. Thanks.